recenter_tc/plugins/modules/output_checkers/fdeck_checker.py

```diff
@@ -40,26 +41,30 @@ def is_fdeck(fname):
 
 
 def test_product_recenter_tc(
-    output_product, compare_product, goodcomps, badcomps, compare_strings
+    plugin, output_product, compare_product, goodcomps, badcomps, compare_strings
 ):
     """Test output_product against "good" product stored in "compare_path".
 
-    Args:
-        output_product
+    Parameters
+    ----------
+    plugin: OutputCheckerPlugin
+        The corresponding geotiff OutputCheckerPlugin that has access to needed methods
     """
```

Comment:
Can this just be named "test_product" and then automatically call "test_products' from the compare_outputs method on the OutputCheckers interface ?  Rather than passing in "test_product_recenter_tc" to compare_outputs?

So, does that mean
* correct_type
* test_product
are additional required methods on output_checkers plugins ?

Reply:
Resolving.